databases/schemas/roomlogs.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE public.roomlogs (
+	type STRING NOT NULL,
+	room STRING NOT NULL,
+	userid STRING NULL,
+	day STRING NOT NULL,
+	month STRING NOT NULL,
+	year STRING NOT NULL,
+	timestamp INT NOT NULL,
```

Comment:
I'm pretty sure our databases consistently call this column `time`. At least, the ones I made do.